pkg/core/blockchain.go

```diff
@@ -504,6 +504,15 @@ func (bc *Blockchain) persist() error {
 			"blockHeight":     bHeight,
 			"took":            time.Since(start),
 		}).Info("blockchain persist completed")
+
+		BCMetrics := &BlockChainMetrics{}
+		BCMetrics.Update(
+			bHeight,
```

Comment:
This one should also be updated in `storeBlock`

Reply:
added there